.gitattributes

```diff
@@ -1 +1,18 @@
-bench/scalajs-runtime-sourcemap.js binary
\ No newline at end of file
+dist/source-map.debug.js binary
```

Comment:
Is the only reason for this so the github interface doesn't show it? I don't really think it's worth it; it would make conflicts harder to figure out, which is more likely with the dist files committed.